src/views/PropTableView.vue

```diff
@@ -250,7 +256,16 @@ function getChildren() {
                     child.link = q.object.value;
                 } else if (q.predicate.value === qname("a")) {
                     child.type = q.object.value;
-                }
+                } else if (item.value.type === qname("dcat:Catalog") && q.predicate.value === qname("dcterms:publisher")) {
+                    const publisher: ListItemSortable = { label: q.object.value };
+                    child.extras.publisher = publisher;
+                } else if (item.value.type === qname("dcat:Catalog") && q.predicate.value === qname("dcterms:creator")) {
+                    const creator: ListItemSortable = { label: q.object.value };
+                    child.extras.creator = creator;
```

Comment:
Are we able to retrieve the label of the object here as well and store it as an object with `iri` and `label`? I think it should display as a hyperlink in the same way as how the vocab listing table shows the reg status and the derivation mode.

Reply:
Added in latest commit